pika/adapters/tornado_connection.py

```diff
@@ -72,7 +73,10 @@ def _adapter_disconnect(self):
         """Disconnect from the RabbitMQ broker"""
         if self.socket:
             self.ioloop.remove_handler(self.socket.fileno())
-        super(TornadoConnection, self)._adapter_disconnect()
+        try:
+            super(TornadoConnection, self)._adapter_disconnect()
+        except exceptions.AMQPConnectionError as e:
```

Comment:
@vitaly-krugl , thanks for checking. The commit message had the reason:

>  tornado_connection: silence AMQPConnectionError in _adapter_disconnect
> It's not possible to handle this exception anywhere and it's super noisy.
18de5f8fcf1c25877522e3ed37404d13795fbc60

AFAICT, if a ```AMQPConnectionError``` happens at that point inside a tornado callback, you as a user can't catch it anywhere reasonably well and it.

However - I recently came across the following messages which seem to be caused by this change (in the scenario of #525 ):
```
CRITICAL:pika.adapters.base_connection:Tried to handle an error where no error existed
```
Seems some other code is relying for the exception to happen so it's not executed... :-/